### PR TITLE
Feat/1271 lnurwithdraw popup

### PR DESCRIPTION
**Pull request by escapedcat**
### Describe the changes you have made in this PR

lnurlWithdraw support via the "Send" within the popup

### Link this PR to an issue

#1271

### Type of change (Remove other not matching type)

- `feat`: New feature (non-breaking change which adds functionality)

### Screenshots of the changes (If any)

_Add screenshots to help explain your problem_

### How has this been tested?

manually with fiatjefs playground

### Checklist

- [x] My code follows the style guidelines of this project and performed a self-review of my own code
- [x] New and existing tests pass locally with my changes
- [x] I checked if I need to make corresponding changes to the documentation (and made those changes if needed)

### CODE REVIEW

**Review by bumi — COMMENTED** [review]
utACK. looks good to me

File: src/types.ts [hunk 1]
```diff
@@ -308,6 +308,8 @@ export interface LNURLWithdrawServiceResponse {
   callback: string; // The URL which LN SERVICE would accept a withdrawal Lightning invoice as query parameter
   k1: string; // Random or non-random string to identify the user's LN WALLET when using the callback URL
   defaultDescription: string; // A default withdrawal invoice description
+  balanceCheck?: string;
```
Comment (escapedcat) [inline src/types.ts:311]: Not sure about this but jef has these

Comment (bumi) [reply src/types.ts:311]: check out the lnurl specs. there are more options that the service can support. 
but we currently don't support the balance check option.

same as paylink I think.

but good to have them in the type anyway.

Comment (escapedcat) [reply src/types.ts:311]: I looked at this: https://github.com/fiatjaf/lnurl-rfc/blob/luds/03.md

**Comment by bumi** [comment]
can you fix the merge conflict and merge?

**Pull request merged**